template.distillery/mobile!eliom_loader.ml

```diff
@@ -91,35 +111,53 @@ let _ =
     (Dom_html.Event.make "resume")
     (Dom.handler (fun _ ->
        log "Resume";
-                    ignore (Js.Unsafe.global##.chcp##fetchUpdate);
-                    Js._true))
+       ignore (Js.Unsafe.global##.chcp##fetchUpdate);
+       Js._true))
     Js._false;
-  let wait2, wake2 = Lwt.wait () in
+  (* Create two threads for success callbacks and error callbacks. *)
+  let wait_success, wake_success = Lwt.wait () in
+  let wait_error, wake_error = Lwt.wait () in
+  (* Callback when success.
+   * [callback ev] will print the event if debug mode is activated.
+   * Calls by the event chcp_nothingToUpdate.
+   *)
   let callback ev =
-    Dom.handler
-      (fun _ ->
-log ev;
-         update_failed := false;
-         Lwt.wakeup wake2 ();
-         Js.bool true)
+    Dom.handler (fun _ ->
+        log ev;
+        update_failed := false;
+        Lwt.wakeup wake_success ();
+        Js._true)
   in
-  List.iter
-    (fun ev ->
-       ignore @@
-       Dom.addEventListener Dom_html.document (Dom_html.Event.make ev)
-         (callback ev) Js._false)
-    ["chcp_nothingToUpdate"];
-  let wait, wake = Lwt.wait () in
```

Comment:
I am happy about the nicer names (`wait_success` and friends), but please refrain from moving things around without good reasons :). Reviewing changes just to make sure that the functionality has stayed the same is not a productive use of time.